packages/gnome_themes_standard.rb

```diff
@@ -1,6 +1,6 @@
-require 'package'
+require 'buildsystems/autotools'
 
-class Gnome_themes_standard < Package
+class Gnome_themes_standard < Autotools
   description 'Standard Themes for GNOME Applications.'
   homepage 'https://git.gnome.org/browse/gnome-themes-standard/'
   version '3.27.92-0'
```

Comment:
Update to [3.28](https://github.com/GNOME/gnome-themes-extra/releases/tag/3.28)?